vars/pipelineVars.groovy

```diff
@@ -52,6 +52,7 @@ class pipelineVars implements Serializable {
     String quayBaseUri = 'quay.io/cloudservices'
     String quayUser = "cloudservices+push"
     String quayPushCredentialsId = "quay-cloudservices-push-token"
+    String stageSS0Url = "https://sso.stage.redhat.com/auth/realms/redhat-external/protocol/openid-connect/token"
```

Comment:
Can you use a the letter "O" instead of number "0" here? :) It is currently `stageSS0Url`